lib/datadoge.rb

```diff
@@ -1,51 +1,13 @@
-require 'datadoge/version'
-require 'datadoge/metrics'
```

Comment:
btw, I think we do need this require to make sure the project using this gem knows about that class. I could be wrong, but if you run into that issue, this might be the cause.